apps/cowswap-frontend/src/modules/twap/containers/TwapFormWarnings/warnings/FallbackHandlerWarning.tsx

```diff
@@ -81,10 +82,10 @@ export function FallbackHandlerWarning({
     return (
       <Wrapper>
         <InlineBannerWithCheckbox bannerType="alert">
-          <strong>Unsupported Safe detected</strong>
+          <strong>Your Safe needs a modification</strong>
           <p>
-            Connected Safe lacks required fallback handler. Switch to a compatible Safe or modify fallback handler for
-            TWAP orders when placing your order.
+            TWAP orders require a one-time update to your Safe to enable signing of conditional (e.g. time-based)
+            orders. It allows them to be automatically executed in the future, once the original TWAP is approved.
```

Comment:
One suggestion would be:

```suggestion
            TWAP orders require a one-time update to your Safe to enable programmatic orders.  These advanced orders allow you to automate your trading strategy with on-chain guarantees. For more information, see [Programmatic orders](https://cow.fi/learn/introducing-the-programmatic-order-framework-from-cow-protocol).
```